awxkit/awxkit/cli/options.py

```diff
@@ -110,7 +113,14 @@ def build_list_actions(self):
                 add_output_formatting_arguments(parser, {})
 
     def build_detail_actions(self):
-        for method in ('get', 'modify', 'delete'):
```

Comment:
We shouldn't just assume that all detail actions just allow these; many actually prevent them for business-logic reasons.